release-notes/5.26.0.md

```diff
@@ -21,6 +21,12 @@ Released June 3, 2020
 | Introduce features?                                             | **yes** |
 | Fix bugs?                                                       | **yes** |
 
+## NOTICE: WordPress Service URLs
+
+The upcoming release of WordPress 5.5 (ETA July 2016) requires changes to several CiviCRM URLs. CiviCRM 5.26+ adopts a new URL convention. For old URLs, there is transitional support that will work for most web-based visitors. However, this may not be sufficient for *external service integrations* (e.g. certain payment processors or email relays which send notifications to CiviCRM).
```

Comment:
Can we change July 5th to August 11th?  Based on the WP 5.5 roadmap the target is August 11th

Reply:
And maybe not 2016 ;-)

Reply:
Or we could just rollback everything to 2016.